components/spin/index.en-US.md

```diff
@@ -37,6 +38,7 @@ Common props ref：[Common props](/docs/react/common-props)
 | spinning | Whether Spin is visible | boolean | true |
 | tip | Customize description content when Spin has children | ReactNode | - |
 | wrapperClassName | The className of wrapper when Spin has children | string | - |
+| fullscreen | Display a backdrop with the `Spin` component | boolean | false |
```

Comment:
You need to add the api version:

```suggestion
| fullscreen | Display a backdrop with the `Spin` component | boolean | false | 5.10.0 |
```

Reply:
Now it seems that it should be landed in version 5.11.0.